portafly/src/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import './index.css'
+import { App } from '@src'
```

Comment:
`yarn start` fails to compile with:

> Failed to compile.
> ./src/index.tsx
> Module not found: Can't resolve '@src' in '/Users/damian/Documents/3scale/porta/portafly/src'

Reply:
It happens on my other laptop, will try to see what's wrong and get back at you

Reply:
ok, now it should be fixed. Take a look
thanks!